curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,527 @@ Which of the following is the default value of the `position` property?
 
 `static`
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which CSS property can be used to hide an element without removing it from the document flow?
+
+#### --distractors--
+
+`display: none`
+
+---
+
+`position: absolute`
+
+---
+
+`z-index: -1`
+
+#### --answer--
+
+`visibility: hidden`
+
+### --question--
+
+#### --text--
+
+Which CSS property allows you to offset an element from its normal position without removing it from the document flow?
+
+#### --distractors--
+
+`absolute`
+
+---
+
+`fixed`
+
+---
+
+`static`
+
+#### --answer--
+
+`relative`
+
+### --question--
+
+#### --text--
+
+What happens to an element with `position: absolute`?
+
+#### --distractors--
+
+It remains in the normal document flow.
+
+---
+
+It sticks to the viewport when scrolling.
+
+---
+
+It overlaps other elements without any control.
```

Comment:
wrong answers shouldn't be noticeably shorter then the correct one.
otherwise, it makes it to easy to guess on the correct one.
https://contribute.freecodecamp.org/how-to-work-on-quizzes/#guidelines-for-designing-questions-and-answers

Please update these so the length is similar to correct answer length